tools/pangolin/pangolin.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="pangolon" name="Pangolin" version="@VERSION@+galaxy0">
+    <description> CoV-2019 Phylogenetic Assignment of Named Global Outbreak LINeages</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+
+pangolin '$input1' -t "\${GALAXY_SLOTS:-1}" --max-ambig '$additional.max_ambig' --min-length '$additional.min_length'
+
+    ]]></command>
+    <inputs>
+        <param name="input1" type="data" format="fasta"
+            label="Input multi-sequence fasta file"
+            />
+        <section name="additional" title="Additional Parameters" expanded="false">
+            <param name="max_ambig" type="float" value="0.5" min="0" max="1.0"
+                label="Maximum proportion of Ns allowed to attempt assignment"
+                argument="--max-ambig"
+                />
+            <param name="min_length" type="integer" value="10000" min="0"
+                label="Minimum query length to attempt assignment"
+                argument="--min-length"
+                />
+        </section>
+    </inputs>
+    <outputs>
+        <data format="csv" from_work_dir="lineage_report.csv" name="lineage_report" label="${input1.name}_lineage_report" />
```

Comment:
can we output a "tabular" output here ... and convert the csv to tab as a second command after pangolin is called?

This way users do not need to convert it with a second tool and get all the tabular features of Galaxy.

Reply:
Yes I will do that as well. I just have to specify tabular and it will auto convert correct

Reply:
No, this is just metadata. You need to do this as part of the tool I would recommend, e.g. with `sed 's/,/\t/g'`.

What you are suggesting is possible, but Galaxy would spin up a second job and this needs to be scheduled and so on ... so its more efficient if we do this in one job.

Reply:
Sounds good, I'll add it that way! Thanks!